src/PrimaryAuthController.js

```diff
@@ -139,10 +139,11 @@ function (Okta, PrimaryAuthForm, SocialAuth, PrimaryAuthModel, Util, BaseLoginCo
 
       this.addListeners();
 
-      // Add SocialAuth view only when the idps are configured. If configured, 'socialAuthPositionTop'
-      // will determine the order in which the social auth and primary auth are shown on the screen.
-      if (options.settings.get('socialAuthConfigured')) {
-        this.add(SocialAuth, {prepend: options.settings.get('socialAuthPositionTop')});
+      // If social auth is configured, 'socialAuthPositionTop' will determine
+      // the order in which the social auth and primary auth are shown on the screen.
+
```

Comment:
nit: remove this newline (i.e. comment should be attached to the code it's talking about)